lib/src/layer/polygon_layer/polygon_layer.dart

```diff
@@ -235,7 +241,7 @@ class PolygonPainter extends CustomPainter {
         }
       }
 
-      if (polygonLabels && polygon.label != null) {
+      if (polygonLabels && !drawLabelsLast && polygon.label != null) {
```

Comment:
I'd also be happy to just remove this entire block, i.e. the old behavior. I'm not sure what's the use case for drawing polygons over labels with labels sometimes sticking out partially. Or appearing faint when overdrawn by semi-transparent polygons.

Reply:
Personally, I think the old behaviour might help with depth perception, so I think it would be a good idea to keep it.

Reply:
Maybe. Maybe if the labels weren't text.

I found it to look like a car crash at best. At worst, it's an accessibility nightmare. Nothing better than a label you wanna read with lines and fills running over it :)

Reply:
> At worst, it's an accessibility nightmare.

Absolutely true of course. Better to colour the text like GitHub does with it's issue labels.

But for the sake of not spoiling anyone's fun, let's just keep it.